modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_contact.yml

```diff
@@ -0,0 +1,46 @@
+id: az_paragraph_contact
+label: AZ Pragrapgh Contact
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: d7_field_collection_item
+  field_name: field_uaqs_contacts
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_contact
+
+process:
+
+  field_az_email:
+    plugin: sub_process
+    source: field_uaqs_contact_email
+    revision_id: revision_id
+    process:
+      delta: delta
+      value: email
+      format: format
+  
+  field_az_phone:
+    plugin: sub_process
+    source: field_uaqs_contact_phone
+    process:
+      delta: delta
+      value: value
+
+  field_az_title:
+    plugin: sub_process
+    source: field_uaqs_contact_name
+    process:
+      delta: delta
+      value: value
+    revision_id: revision_id
```

Comment:
I think we want to omit the revision_id mappings in these migrations as discussed in our PR meeting yesterday.
```suggestion
```